client/src/pages/Login.jsx

```diff
@@ -74,28 +97,36 @@ function Login(props: Props) {
             rules: [
               {
                 required: true,
-                message: 'Please input your password!',
+                message: intl.formatMessage(messages.pleaseInputYourPassword),
               },
             ],
           })(
             <Input
               prefix={<LockOutlined style={{ color: 'rgba(0,0,0,.25)' }} />}
               type="password"
-              placeholder="Password"
+              placeholder={intl.formatMessage(messages.password)}
             />,
           )}
         </Form.Item>
         <div className="red-text">{error}</div>
         <Form.Item>
           <Button type="primary" htmlType="submit" className="form-button">
-            Log In
+            <FormattedMessage id="LogIn" defaultMessage="Log In" />
```

Comment:
I'm wondering if our db is case sensitive with IDs or not - just to be safe, could we make this id something like `logInButton`